my-page-app/src/components/ui/monthCalendar.tsx

```diff
@@ -209,8 +181,11 @@ function MonthCalendar({
               {dateCalendar}
               {cabinBookings}
               <span
-                  className={`absolute top-0 left-0 w-full h-full ${getCutOffDateStyle(props.date, cutOffDate)}`}
-                  aria-hidden="true"
+                className={`absolute top-0 left-0 w-full h-full ${getCutOffDateStyle(
+                  props.date,
+                  cutOffDate,
+                )}`}
```

Comment:
Vi har en utility classe som gjør conditional styling litt cleanere:

```suggestion
                className={
                    cn("absolute top-0 left-0 w-full h-full", 
                          getCutOffDateStyle(
                              props.date,
                              cutOffDate,
                          )
                      )
                  }
```